src/main/java/bio/terra/cloudfunctions/utils/MediaTypeUtils.java

```diff
@@ -0,0 +1,68 @@
+package bio.terra.cloudfunctions.utils;
+
+import com.google.cloud.ReadChannel;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageException;
+import com.google.cloud.storage.StorageOptions;
+import java.io.BufferedInputStream;
+import java.io.InputStream;
+import java.nio.channels.Channels;
+import org.apache.commons.compress.archivers.ArchiveException;
+import org.apache.commons.compress.archivers.ArchiveInputStream;
+import org.apache.commons.compress.archivers.ArchiveStreamFactory;
+import org.apache.commons.compress.compressors.CompressorException;
+import org.apache.commons.compress.compressors.CompressorInputStream;
+import org.apache.commons.compress.compressors.CompressorStreamFactory;
+
+public class MediaTypeUtils {
+  /**
+   * If a Service Account has not been specified for Google Cloud Function deployment, then the
+   * Cloud Function will assume the roles of the default IAM Service Account
+   * PROJECT_ID@appspot.gserviceaccount.com at runtime.
+   *
+   * <p>The Service Account for any Java 11 runtime (Compute Engine, App Engine, or GKE) must have
+   * appropriate GCS read permissions.
+   *
+   * <p>Opens an input stream to GCS Object.
+   *
+   * @param projectId the Google Project ID
+   * @param bucket the Google Storage Bucket
+   * @param objectName the Google Storage Bucket File Object
+   * @return URL object
+   */
+  public static InputStream getStorageObjectDataAsInputStream(
```

Comment:
I feel like this method belongs in a different class since it involves calling Google APIs and fetching a file from a bucket whereas the other two functions are more general I/O.

Reply:
Pushed an update. I have moved `getStorageObjectDataAsInputStream` to a new class called `GcsUtils`.